cleanlab/datalab/internal/issue_manager/regression/label.py

```diff
@@ -64,7 +64,7 @@ def __init__(
         self,
         datalab: Datalab,
         clean_learning_kwargs: Optional[Dict[str, Any]] = None,
-        threshold: float = 0.1,
+        threshold: float = 0.05,
```

Comment:
Why is this change here? If default is now 0.05, update the docstring for `RegressionLabelIssueManager` above with new value^

Reply:
The label quality score for regression is based on the [outer score](https://docs.cleanlab.ai/stable/cleanlab/regression/rank.html?highlight=outre#cleanlab.regression.rank.get_label_quality_scores) (basically the feature-based OOD score from `OutOfDistrubtion`).

The way label issues are flagged in this case is:

```python
label_quality_score: np.ndarray
is_label_issue = label_quality_score < threshold * np.median(label_quality_score)
```

The distribution of the `label_quality_score`  is affected by this PR. I've noticed that the normalization step causes the label quality scores to spread out more than the previous score.
A lower threshold is necessary for a similar "rate" of expected issues.